mlflow/tracking/_model_registry/fluent.py

```diff
@@ -701,11 +702,14 @@ def load_prompt(
         name_or_uri: The name of the prompt, or the URI in the format "prompts:/name/version".
         version: The version of the prompt (required when using name, not allowed when using URI).
         allow_missing: If True, return None instead of raising Exception if the specified prompt
-            is not found.
+            is not found. Note: If ``fallback`` is provided, this is automatically set to True.
         link_to_model: If True, the prompt will be linked to the model with the ID specified
                        by `model_id`, or the active model ID if `model_id` is None and
                        there is an active model.
         model_id: The ID of the model to which to link the prompt, if `link_to_model` is True.
+        fallback: The fallback prompt to use if the prompt loading fails. It follows the same format
```

Comment:
Does `loading fails` mean "any failure"? In LangFuse, when is the fallback used?